database/models/events.db.php

```diff
@@ -100,10 +100,6 @@ class Event
     #[OneToMany(targetEntity: EventEntry::class, mappedBy: 'event', cascade: ["remove"])]
     public Collection $entries;
 
-    /** @var Collection<int, Race> */
-    #[OneToMany(targetEntity: Race::class, mappedBy: 'event', cascade: ["remove"])]
-    public Collection $races;
-
```

Comment:
Same here: revert this change (see previous comment)

Reply:
✅